src/main/java/io/vinyldns/java/model/membership/UserInfo.java

```diff
@@ -0,0 +1,106 @@
+package io.vinyldns.java.model.membership;
+
+import io.vinyldns.java.model.zone.Zone;
+import org.joda.time.DateTime;
+
+public class UserInfo {
```

Comment:
`UserInfo` is based off of `MembershipProtocol` from VinylDNS (see: https://github.com/vinyldns/vinyldns/blob/202dcebba406d4a992e1d000e164422ee43dd5ba/modules/api/src/main/scala/vinyldns/api/domain/membership/MembershipProtocol.scala#L71-L79).